hadoop-ozone/dist/src/main/smoketest/s3/MultipartUpload.robot

```diff
@@ -406,3 +415,84 @@ Test Multipart Upload list
 
     ${count} =          Execute and checkrc      echo '${result}' | jq -r '.Uploads | length'  0
                         Should Be Equal          ${count}     2
+
+*** Variables ***
+${ENDPOINT_URL}       http://s3g:9878
+${BUCKET}             generated
+${BUCKET1}            generated
```

Comment:
Could we improve the naming of variables here? BUCKET1 is very ambiguous, it can be changed to BUCKET_FSO or anything similar.

Reply:
+1